src/WattleScript.Interpreter/Debugging/SourceRef.cs

```diff
@@ -259,17 +263,22 @@ internal void WriteBinary(BinDumpWriter writer)
 			writer.WriteVarInt32(ToChar - FromChar);
```

Comment:
This can be done in another PR if needed but it is probably worth now having a test to make sure that SourceRefs are being serialized correctly as we've long since gone past just dumping the struct (which is good for space usage).